examples/block-explorer/explorer-index/src/lib.rs

```diff
@@ -21,6 +21,7 @@
 //! cargo run --bin fuel-indexer -- --manifest examples/block-explorer/manifest.yaml
 //! ```
 
+use fuels_core::tx::field::*;
```

Comment:
What is this?

Reply:
@deekerno  
- Ah got, it. So for this -- and for any time that you need to pull in various types from the rest of the project into the macros we go through `fuel-indexer-plugin`
- So for this, I would either:
  1. Add it to `fuel-indexer-plugin::types`
  2. Since `fuel-indexer-plugin::types` just pulls from the new `fuel-indexer-types` we could add it this `fuels_core::tx::field::*` there
    - And if it's related to transactions it would go in `fuel-indexer-types::tx`